tools/build_defs/cc/cc_import.bzl

```diff
@@ -200,16 +184,14 @@ def _cc_import_impl(ctx):
         name = ctx.label.name,
     )
 
-    if output_file:
-        _create_archive_action(ctx, feature_configuration, cc_toolchain, output_file, object_files)
-
     return [CcInfo(compilation_context = compilation_context, linking_context = linking_context)]
 
 cc_import = rule(
     implementation = _cc_import_impl,
     attrs = {
         "hdrs": attr.label_list(allow_files = [".h"]),
-        "static_library": attr.label(allow_single_file = [".a", ".lib", ".pic.a"]),
+        "static_library": attr.label(allow_single_file = [".a", ".lib"]),
+        "pic_static_library": attr.label(allow_single_file = [".a", ".lib", ".pic.a"]),
```

Comment:
These should be .pic.a and .pic.lib:
https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/rules/cpp/CppHelper.java;drc=8b856f5484f0117b2aebc302f849c2a15f273310;l=459